.github/workflows/test.yml

```diff
@@ -20,16 +44,6 @@ jobs:
         uses: ./.github/actions/with-docker
         with:
           container-name: mir-semantics-ci-${{ github.sha }}
-      - name: 'Install dependencies'
-        run: docker exec --user user mir-semantics-ci-${GITHUB_SHA} poetry -C kmir install
-      - name: 'Check code quality'
-        run: docker exec --user user mir-semantics-ci-${GITHUB_SHA} make -C kmir check
-      - name: 'Run unit tests'
-        run: docker exec --user user mir-semantics-ci-${GITHUB_SHA} make -C kmir test-unit
-      - name: 'Build LLVM semantics'
-        run: docker exec --user user mir-semantics-ci-${GITHUB_SHA} make -C kmir kbuild-llvm
-      - name: 'Build Haskell semantics'
-        run: docker exec --user user mir-semantics-ci-${GITHUB_SHA} make -C kmir kbuild-haskell
```

Comment:
I dropped these, `kbuild kompile` is checked as part of `test-integration` anyways.